packages/vue_frontend_rpc/test/functional/initial-data-visualization.spec.js

```diff
@@ -1,20 +1,16 @@
 import { expect } from "chai";
 import { mount } from "@vue/test-utils";
-const getPort = require("get-port");
-
 import PersonForm from "@/components/person-form.vue";
-import { BackendMock } from "../../local-dev/backend-mock";
+import { backendMock } from "../backend-mock";
+import flushPromises from "flush-promises";
 
 describe("The Person Form", () => {
   describe("initial data visualization flow", () => {
-    let backendMock;
-    let port;
+    let mockInvokeObject;
+    let mountOptions;
 
     before(async () => {
-      port = await getPort({ port: getPort.makeRange(9000, 10000) });
-      // Start an in-memory websocket backend mock for this test.
-      backendMock = new BackendMock({
-        port: port,
+      mockInvokeObject = backendMock({
```

Comment:
I remember we were discussing about it when I wrote the tests in the welcome page.. I couldn't make it work with the websocket approach setup so then you suggested the `backendMock` for tests that will return the `invokeMock` function with the custom implementation. 
I guess it is a matter of how much time you put into investigate it vs the value it will bring to the tests 
We can discuss on it/take a look together if you think it is important.